src/codemodder/codemods/codeql.py

```diff
@@ -0,0 +1,29 @@
+from functools import cache
+from pathlib import Path
+
+from codemodder.codemods.base_detector import BaseDetector
+from codemodder.codeql import CodeQLResultSet
+from codemodder.context import CodemodExecutionContext
+from codemodder.result import ResultSet
+
+
+class CodeQLSarifFileDetector(BaseDetector):
+    def apply(
+        self,
+        codemod_id: str,
+        context: CodemodExecutionContext,
+        files_to_analyze: list[Path],
+    ) -> ResultSet:
+        del codemod_id
+        del files_to_analyze
```

Comment:
What is the significance of del here?

Reply:
It makes the type checker happy by indicating that these variables are unused.